docs_dev/releasing.md

```diff
@@ -181,78 +179,6 @@ Identify the [release](https://github.com/kubeflow/pytorch-operator/releases) of
 Update [pytorch-operator.jsonnet](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/pytorch-job/prototypes/pytorch-operator.jsonnet#L9)
 to point to the new image.
 
-## Build TF Serving Images
-
-We build TF serving images using an argo workflow.
-
-Look at the [postsubmit dashboard](https://k8s-testgrid.appspot.com/sig-big-data#kubeflow-postsubmit)
-to find the latest green postsubmit.
-
-Check out that commit
-
-```
-COMMIT=<commit to build>
-cd ${GIT_KUBEFLOW}
-git checkout ${COMMIT}
-cd releasing/releaser
-```
-
-```
-PULL_BASE_SHA=${COMMIT:0:8}
-DATE=`date +%Y%m%d`
-VERSION_TAG="v${DATE}-${PULL_BASE_SHA}"
-JOB_NAME="tf-serving-release"
-JOB_TYPE=postsubmit
-BUILD_NUMBER=$(uuidgen)
-BUILD_NUMBER=${BUILD_NUMBER:0:4}
-REPO_OWNER=kubeflow
-REPO_NAME=kubeflow
-ENV=releasing
-ks param set --env=${ENV} workflows namespace kubeflow-releasing
-ks param set --env=${ENV} workflows name "${USER}-${JOB_NAME}-${VERSION_TAG}"
-ks param set --env=${ENV} workflows prow_env "JOB_NAME=${JOB_NAME},JOB_TYPE=${JOB_TYPE},PULL_BASE_SHA=${PULL_BASE_SHA},REPO_NAME=${REPO_NAME},REPO_OWNER=${REPO_OWNER},BUILD_NUMBER=${BUILD_NUMBER}"
-ks param set --env=${ENV} workflows versionTag ${VERSION_TAG}
-ks apply ${ENV} -c workflows
-```
-
-## Build the Jupyter Images
-
-We build the Jupyter using our E2E argo workflows.
```

Comment:
How do we release jupyter images? Its possible the postsubmits are automatically pushing to the public repo but I'm not sure.

Reply:
yes, it does.  we confirmed:
https://github.com/kubeflow/manifests/commit/723f310a60c9765bcadc2ed75053d5819f429c60